spec/openapi.yaml

```diff
@@ -1941,9 +1978,21 @@ paths:
         - Source Address
       summary: Delete Sender Address
       description: |
-        Delete a sender address
+        The Delete Sender Address endpoint allows you to remove any registered sender addresses that are no longer required from your account.
 
-        This endpoint allows you to remove any Sender Addresses that are no longer required.
+        The following parameters are required on your request:
+          - ```id:``` **(Required)**. The UUID of the sender address
+          - ```reason:``` **(Required)**. A string detailing why the sender address is being removed
+        
+        **Example request with Delete a Sender Address**
```

Comment:
Regarding the example, is it necessary or redundant? This is because with OpenAPI, if everything is defined correctly, the request and response examples are already provided.